jib-cli/src/test/java/com/google/cloud/tools/jib/cli/jar/JarFilesTest.java

```diff
@@ -217,4 +237,34 @@ public void testToJibContainerBuilder_packagedSpringBoot_basicInfo()
                 .build()
                 .getEntries());
   }
+
+  @Test
+  public void testToJibContainerBuilder_dockerBaseImage()
+      throws IOException, InvalidImageReferenceException {
```

Comment:
Currently, I can't think of a good way to test for the tar base image without accessing the build context (similar to what's being done in PluginConfigurationProcessorTest)

Reply:
Ah, you mean `ContainerBuildPlan.getBaseImage()` is just an image reference without `docker://` or `tar://`, so it's not possible to check if `JibContainerBuilder` was correctly configured, right?

One way (which I don't really like) is to create an [indirection class to expose a package-protected method](https://github.com/GoogleContainerTools/jib/blob/master/jib-plugins-common/src/test/java/com/google/cloud/tools/jib/api/JibContainerBuilderTestHelper.java), which is what `PluginConfigurationProcessorTest` did. The helper is under the `c.g.c.t.jib.api` package, so it can access `toBuildContext()`.

Reply:
Ah yes, exactly. Thank you! Modified the tests with this solution.